.eslintignore

```diff
@@ -1,6 +1,5 @@
 node_modules
 bower_components
-lib
```

Comment:
question: why this change?

Reply:
Have a lib directory in the docs package now for holding things like google analytics (to match practices from the [next official examples](https://github.com/vercel/next.js/tree/canary/examples/with-google-analytics)).